src/main/java/uk/gov/ons/ctp/response/action/service/ProcessEventService.java

```diff
@@ -720,6 +721,7 @@ private void processEmail(
                 .personalisation(personalisation)
                 .classifiers(classifiers)
                 .emailAddress(respondentParty.getAttributes().getEmailAddress())
+                .reference(survey.getSurveyRef() + "-" + businessParty.getAttributes().getRuref())
```

Comment:
Is it possible for either of these to be blank?  I don't think so but I figured I'd ask.

Reply:
Not as far as I'm aware, and if somehow either was I'm not sure how we'd even handle that